qa/tasks/ceph_objectstore_tool.py

```diff
@@ -511,7 +511,7 @@ def test_objectstore(ctx, config, cli_remote, REP_POOL, REP_NAME, ec=False):
                 try:
                     info = remote.sh(cmd, wait=True)
                 except CommandFailedError as e:
-                    log.error("Failure of --op info command with {ret}".
+                    log.error("Failure of --op info command with %s".
                               format(e.exitstatus))
```

Comment:
The format is unused here now

Reply:
? it's not. There is `%s` . Or am I missunderstanding your comment?

Reply:
format does not use %s, you use either:
```
log.error("Failure of --op info command with {}".format(e.exitstatus)) 
```
or
```
log.error("Failure of --op info command with %s", e.exitstatus) 
```

Reply:
@tchaikov still forgot to remove _format_.

Reply:
ah. true. thx

Reply:
```
In [1]: format(1)
Out[1]: '1'
```

Reply:
> ```
> In [1]: format(1)
> Out[1]: '1'
> ```

Any sense of doing it?

Reply:
no, JFYI.